### ISSUE TITLE
told to "boot with new key" instead of using |rekey when rotating keys without breaching

### ISSUE THREAD

**Issue by drbeefsupreme** [urbit/bridge#868]
When you rotate keys without breaching, you need to import the new keys into a running ship, not boot a new one. Right now, Bridge tells you to boot with the new keys, when it should tell you to use `|rekey`. `|rekey` doesn't work with the new key format right now, but I fixed it yesterday so it will be in production soon.

**Comment by nerveharp** [comment]
![image](https://user-images.githubusercontent.com/748181/150820310-a95a3278-72f0-4457-839a-0c8cb2dc6b11.png)

**Comment by drbeefsupreme** [comment]
https://github.com/urbit/urbit/pull/5522

**Comment by nerveharp** [comment]
I eliminated the "Download Keyfile" button in this instance, because if what we are saying is true, you don't _actually_ need your keyfile after re-keying.

![image](https://user-images.githubusercontent.com/748181/150847061-368b00c1-a7e7-4ffa-bd68-0ef7d93dd951.png)

@drbeefsupreme @e-shee Comments welcome.

**Comment by drbeefsupreme** [comment]
I don't think that is correct. What you need to do is copy the contents of the keyfile, then run `|rekey 'keyfile-contents'`. `|rekey` by itself doesn't do anything, and you wouldn't be able to pull the key from Ethereum since it only has your public key, while your keyfile contains your private key.

**Comment by nerveharp** [comment]
OK, so we will still need the "Download Keyfile" button. I will adjust the copy in a second pass.

**Comment by nerveharp** [comment]
![image](https://user-images.githubusercontent.com/748181/150859300-91b1ab61-39af-4ae2-8759-4dba078da742.png)

**Comment by drbeefsupreme** [comment]
LGTM

**Comment by e-shee** [comment]
Copy is clean, lgtm.

**Comment by Fang-** [comment]
>bootkey

Where did this word come from? The latest screenshot posted seems doubly confusing because it says "boot" but then doesn't actually instruct me to boot anything, to use my already-running ship instead? imo this is a strict downgrade from "keyfile".

(Not letting you download if you didn't breach is nice though. But we must be aware that this means the user cannot easily save the key for later use, if they do not have access to their dojo right this instant.)

**Comment by nerveharp** [comment]
@Fang- This comes from an effort between @e-shee, @jyng and I to optimize the language in the claiming / onboarding flow. It's a compound verb-noun, one "boots" with the "key." It instructs the user what the file is for, rather than relying on supplemental instructions "boot the ship with your keyfile."

I'm happy to go around and around on this, but please understand that we are keeping an eye towards walk-up usability.
